LondonBritishLibrary/orient/BLorient9798.xml

```diff
@@ -0,0 +1,281 @@
+                                    <lb n="4"/>ዘመን፡ ለከ፡ እመ፡ ኃለፈ።
+                                    <lb n="5"/>አሐዱ፡ ወልድከ፡ ያስተጋብእ፡ ጽንፈ።
+                                    <lb n="6"/>ወክልኤቱ፡ ይስድዱ፡ እልፈ።</incipit>
+                            </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="50r" to="56v"/>
+                            <title ref="LIT0000000000000000000">Calendar of the movable feasts (Ḥasaba ḥǝgg)</title>
+                            <note>A kind of perpetual calendar based on the great lunar cycle of 532 years. The columns contain: 
+                                (1) <foreign xml:lang="gez">አው፡ ቀመ፡ </foreign>, (2) <foreign xml:lang="gez">ዓው፡ ዐመ፡ </foreign>, 
+                                (3) <foreign xml:lang="gez">ዕጥ፡ ለን፡</foreign>, (4) <foreign xml:lang="gez">አበቅቴ፡ </foreign>, 
+                                (5) <foreign xml:lang="gez">መጥቅዕ፡ </foreign>, (6) <foreign xml:lang="gez">ነነዌ፡ </foreign>, 
+                                (7) <foreign xml:lang="gez">በዓተ፡ ጾም፡</foreign>, (8) <foreign xml:lang="gez">ፍሥሕ፡</foreign>, 
+                                (9) <foreign xml:lang="gez">ፋሲካ፡</foreign>. In the left margin historical notes (<foreign xml:lang="gez">ታሪክ፡</foreign>) 
+                                are added. They seem to be written by at least two different hands at different times. The last note concerns the death of
+                                the Emperor <persName ref="PRS9430Tewodros">Theodore (Tewodros) (1860 EC, i.e. 1868 AD)</persName>.
+                                That event is mentioned under the year '523' based on the Era of the Martyrs, which should however be corrected to
+                                <date when-custom="521" calendar="grace" when="1868">521</date> to correspond to the year of Tewodros' death.
+                            </note>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="57v" to="58r"/>
+                            <title ref="LIT0000000000000000000">Qǝne poems attributed to Gabra Masiḥ</title>
+                            <note>Nine <ref type="authFile" corresp="#Qene">qǝne songs</ref> attributed to person with the name 
+                                <persName ref="PRS14499GabraMasih" role="author">Gabra Masiḥ</persName>: 
+                                5 <ref type="authFile" corresp="#EtanaMogar"><foreign xml:lang="gez-tr">ʿǝṭāna mogar</foreign></ref>, 
+                                1 <ref type="authFile" corresp="#Wazema"><foreign xml:lang="gez-tr">wāzemā</foreign></ref>, 
+                                1 <ref type="authFile" corresp="#Sellase"><foreign xml:lang="gez-tr">śǝllāse</foreign></ref>, 
+                                and 1 in the <foreign xml:lang="gez">ጉባዔ፡ ቃና፡</foreign>-type,
```

Comment:
We should probably create an authority file for the qǝne type Gubāʾe Qānā.

It's one of the types mentioned in lists, for example in the EAe (s.v. qǝne).

Reply:
It is not listed among the seven modes that are listed as "Qene types" in the Qene-related files:
https://betamasaheft.eu/authority-files/list?keyword=Qene
As well, I did not find it, when looking up ጉባዔ፡ or ቃና. So, I think, it does not exist, yet.

Reply:
Yes, then creating it would probably be a good idea. Or what do you say, @DenisNosnitsin1970 ?

Reply:
I found [Gubāʾe qānā](https://betamasaheft.eu/authority-files/Gubaeqana/main?hi=Gubae) as an authority file.